README.md

```diff
@@ -110,41 +110,74 @@ This sample shows how to create custom AnalyzeRules for use within the Analyze w
-   1. Prepare each bundle file to be assigned to its own asset pack:
-      - Each asset pack will have a directory named “{asset pack name}.androidpack” in “Assets/PlayAssetDelivery/CustomAssetPackContent".
-      - Each .androidpack directory also contains a ‘build.gradle’ file that specifies that delivery type for the asset pack. If this file is missing, Unity will assume that the asset pack’s delivery type is “on-demand”.
-   2. Create a "CustomAssetPacksData.json" file in 'Assets/StreamingAssets' that stores all custom asset pack information to be used at runtime.
-8. Prepare Addressables to load content from asset packs at runtime (we do this in AddressablesInitSingleton.cs).
-   1. Make sure that the generated asset packs are downloaded.
-   2. Configure our custom InternalIdTransformFunc, which converts internal ids to their respective asset pack location.
-   3. Load all custom asset pack data from the "CustomAssetPacksData.json".
-   4. Then load assets using the Addressables API (we do this in LoadObject.cs).
-9. When ready to build the Android App Bundle, open File > Build Settings and click “Build”. This will create all of our custom asset packs along with the generated asset packs.
-   - If you want to upload the App Bundle to the Google Play Console, make sure that you are doing a release build.
+
+##### Configure custom Addressable scripts
+The sample project uses custom Addressable scripts that make it easier to build and load AssetBundles assigned to asset packs.
+- In 'Assets/PlayAssetDelivery/Editor' we have:
+  - A custom group template (Asset Pack Content.asset).
+  - A custom data builder script (BuildScriptPlayAssetDelivery.cs) and its serialized asset (BuildScriptPlayAssetDelivery.asset)
+  - A custom build processor (PlayAssetDeliveryBuildProcessor.cs)
+  - A custom schema (PlayAssetDeliverySchema.cs)
+- In 'Assets/PlayAssetDelivery/Runtime' we have:
+  - A custom AssetBundle Provider (PlayAssetDeliveryAssetBundleProvider.cs)
+
+Most of the scripts are automatically imported, but there are a few assets that need to be manually configured in AddressableAssetSettings:
+1. In "Build and Play Mode Scripts" add the custom data builder asset (BuildScriptPlayAssetDelivery.asset).
+2. In "Asset Group Templates" add the custom group template (Asset Pack Content.asset).
+
+##### Assign Addressable Groups to asset packs
+1. In the Addressables Groups window (Window > Asset Management > Addressables Groups) toolbar, select Create > Group > Asset Pack Content to create a new group whose content will be assigned to an asset pack. The group contains 2 schemas “Content Packing & Loading” and "Play Asset Delivery”.
+2. Specify the assigned asset pack in “Play Asset Delivery” schema. Select "Manage Asset Packs" to modify custom asset packs. Be mindful of the [size restrictions per delivery mode](https://developer.android.com/guide/playcore/asset-delivery#custom-asset-packs).
```

Comment:
Remove "be mindful of the [size restrictions per delivery mode](link)" since we already mention this advice further up the page.